src/common/dsp/DPWOscillator.h

```diff
@@ -66,17 +66,17 @@ class DPWOscillator : public Oscillator
 
     // character filter
     bool dofilter = true;
-    double charfiltB0, charfiltB1, charfiltA1;
-    double priorY_L, priorY_R, priorX_L, priorX_R;
+    double charfiltB0 = 0.f, charfiltB1 = 0.f, charfiltA1 = 0.f;
```

Comment:
0.0 not 0.f for doubles.

Reply:
(not that it matters; don't redo the PR for just this. but i will fix it up when clion complains at me :))